asmcli/asmcli.sh

```diff
@@ -246,23 +246,39 @@ gcloud() {
 }
 
 kubectl() {
-  local KCF KCC
+  local KCF KCC HTTPS_PROXY
   KCF="$(context_get-option "KUBECONFIG")"
   KCC="$(context_get-option "CONTEXT")"
+  HTTPS_PROXY="$(context_get-option "HTTPS_PROXY")"
   if [[ -z "${KCF}" ]]; then
     KCF="${KUBECONFIG}"
   fi
+
+  if [[ -n "${HTTPS_PROXY}" ]]; then
+    export "HTTPS_PROXY=${HTTPS_PROXY}"
+  fi
   run_command "${AKUBECTL}" --kubeconfig "${KCF}" --context "${KCC}" "${@}"
+  unset HTTPS_PROXY
```

Comment:
I think we should be able to do one of these and not export anything:

```
"HTTPS_PROXY=${HTTPS_PROXY}" run_command "${AKUBECTL}" --kubeconfig "${KCF}" --context "${KCC}" "${@}"
run_command "HTTPS_PROXY=${HTTPS_PROXY}" "${AKUBECTL}" --kubeconfig "${KCF}" --context "${KCC}" "${@}"
```

Reply:
updated, thanks.